modules/liveIntentIdSystem.js

```diff
@@ -139,9 +138,6 @@ export const liveIntentIdSubmodule = {
   setModuleMode(mode) {
     this.moduleMode = mode
   },
-  getInitializer() {
```

Comment:
This is unnecessary because LiveConnect will check the env: `process.env.LiveConnectMode` that is already set [here](https://github.com/LiveIntent/Prebid.js/blob/cm-610/modules/liveIntentIdSystem.js#L131)

Reply:
Can we test it?

Reply:
I ran `gulp serve --modules=userId,uid2IdSystem,liveIntentIdSystem,liveintentAnalyticsAdapter,rubiconBidAdapter --host=0.0.0.0 --watch` and 
` LiveConnectMode='minimal' gulp serve --modules=userId,uid2IdSystem,liveIntentIdSystem,rubiconBidAdapter --host=0.0.0.0 --watch
` and it worked

Reply:
We expose a function `setModuleMode(mode)` that sets the module mode. After your change that setting will be completely ignored -- I don't think we should go that way / we need to at least communicate this in some sort of changelog

Reply:
Ok I will put it back